src/pages/criteria-page/CriteriaPage.tsx

```diff
@@ -93,6 +94,7 @@ export default function CriteriaPage(props: Props) {
             thoughtLeadershipComment: review?.thoughtLeadershipComment || undefined,
             thoughtLeadershipRating: review?.thoughtLeadershipRating || undefined,
           }}
+          isSelfReview={data.viewer.review?.isSelfReview || false}
```

Comment:
```suggestion
          isSelfReview={review?.isSelfReview}
```
`isSelfReview` is not optional

Reply:
Done